bowtie/_report.py

```diff
@@ -314,8 +314,20 @@ def flat_results(self):
     def generate_badges(self, target_dir: Path, dialect: str):
         label = _DIALECT_URI_TO_SHORTNAME[dialect]
         for impl in self.implementations:
+            dialectVersions = impl["dialects"]
```

Comment:
The convention in Python is different from Javascript, variables are generally named in snake case, so this should be

```suggestion
            dialect_versions = impl["dialects"]
```